src/System.Drawing.Common/pkg/System.Drawing.Common.pkgproj

```diff
@@ -8,8 +8,22 @@
     <ProjectReference Include="..\src\System.Drawing.Common.csproj" />
 
     <InboxOnTargetFramework Include="$(AllXamarinFrameworks)" />
+
     <InboxOnTargetFramework Include="netcoreapp2.1" />
+    <File Include="$(PlaceHolderFile)">
+      <TargetPath>runtimes/win/lib/netcoreapp2.1</TargetPath>
+    </File>
+    <File Include="$(PlaceHolderFile)">
+      <TargetPath>runtimes/unix/lib/netcoreapp2.1</TargetPath>
+    </File>
+
     <InboxOnTargetFramework Include="netcoreapp2.2" />
```

Comment:
No need to add 2.2 stuff. Since it is inbox in 2.1 and it doesn't have any implementations for 2.2, it will take the inbox implementation in 2.2 as well.